network/p2p/peer_tracker.go

```diff
@@ -17,10 +17,9 @@ import (
 	"github.com/ava-labs/avalanchego/utils"
 	"github.com/ava-labs/avalanchego/utils/heap"
 	"github.com/ava-labs/avalanchego/utils/logging"
+	safemath "github.com/ava-labs/avalanchego/utils/math"
 	"github.com/ava-labs/avalanchego/utils/set"
 	"github.com/ava-labs/avalanchego/version"
-
-	safemath "github.com/ava-labs/avalanchego/utils/math"
 )
 
 const (
```

Comment:
Should these `const`s be a config?

Reply:
I think we do want them to be at some point (for example we want to allow people to send app requests either routed entirely randomly or by bandwidth, so we need a way for the caller to tune some of these values).